tests/math_tools/test_partial_derivatives.cpp

```diff
@@ -735,4 +799,13 @@ TEST(PartialDerivative, CentralFDMPartialDerivativeWithBV)
 
     EXPECT_LE(relative_error_order_y, TOL);
 }
+
+TEST(PartialDerivative, ConstantPartialDerivative)
+{
+    const double val(3.5);
+    PartialDerivativeTestConstant<10, 10> const test(val);
```

Comment:
Would be maybe better to test with a different number of cells in x and y ?

Reply:
Not sure it makes much difference for this particular operator but I agree in general
```suggestion
    PartialDerivativeTestConstant<10, 5> const test(val);
```

Reply:
I completely agree with you. It is not specific to this test but I think it would be better to take the habit to choose different values for all directions for tests.